src/components/atoms/Navigator/index.tsx

```diff
@@ -0,0 +1,177 @@
+import { memo } from "react";
+
+import { useT } from "@reearth/i18n";
+import { styled } from "@reearth/theme";
+
+import Icon from "../Icon";
+
+import { useNavigator } from "./hooks";
+import { CompassAngle } from "./types";
+
+export type Props = {
+  degree: number;
+  /**
+   * Pass degree of circle as callback arguments.
+   * This event is invoked when outer circle is clicked and mouse move.
+   * If click is dismissed then this event is stopped.
+   */
+  onRotate?: (degree: number) => void;
+  /**
+   * Pass distance of movement as x,y axis(px) and degree of circle as callback arguments.
+   * This event is invoked when inner circle button is clicked and mouse move.
+   * If click is dismissed then this event is stopped.
+   */
+  onOrbit?: (angle: CompassAngle) => void;
+  /**
+   * This event is invoked when center of circle is clicked.
+   */
+  onRestoreRotate?: () => void;
+  /**
+   * Pass + or -  as callback arguments.
+   */
+  onClickHelp?: () => void;
+  onZoomIn?: () => void;
+  onZoomOut?: () => void;
+};
+
+const Navigator: React.FC<Props> = memo(function NavigatorPresenter({
+  degree,
+  onRotate,
+  onOrbit,
+  onRestoreRotate,
+  onClickHelp,
+  onZoomIn,
+  onZoomOut,
+}) {
+  const {
+    compassRef,
+    compassDegree,
+    compassFocusAngle,
+    isMovingAngle,
+    handleOnMouseDownAngle,
+    handleOnMouseDownCompass,
+  } = useNavigator({ degree, onRotate, onOrbit });
+  const t = useT();
+  return (
+    <Container>
+      <CompassContainer>
+        <Compass ref={compassRef}>
+          <>
+            <CompassIcon onMouseDown={handleOnMouseDownCompass}>
+              <Icon
+                icon="compass"
+                color="#000"
+                aria-label={t("compass")}
```

Comment:
Since aria-label's are unique and seem to not need grammar, in-depth descriptions, etc, can we consider doing `aria-label-someNameOrDescription`?
That way it'll be a bit clearer in the translation files. It DOES mean we have to update with the ACTUAL desired value inside the translation file. So currently, we just copy and paste the key to the value in the English.
Please feel free to disagree with this approach or if you have a better idea please let me know!

Reply:
Does what you mean that we should use the special key for `aria-label` like the following?

use special key

```js
aria-label={t("aria-label-compass")}
```

and set to translation file

```yaml
"aria-label-compass": compass
```

Reply:
Yes! Exactly!

Reply:
https://github.com/reearth/reearth-web/pull/308/commits/ad87c866ed63aa40a82916eb8816e286eb946c52
I fixed this!